Userland/Libraries/LibWeb/Forward.h

```diff
@@ -8,7 +8,6 @@
 #pragma once
 
 #include <LibJS/Forward.h>
-#include <LibWeb/Bindings/Forward.h>
```

Comment:
Will mention that I suppose you still have a rebuild of the world once the actual platformobject is added here, but I think that's okay because at least it happens when it's desired, instead of immediately being forced to rebuild everything by the buildsystem just by changing `idl_files.cmake`